pkg/grpc/dapr/dapr_api_unimplement.go

```diff
@@ -55,7 +59,39 @@ func (d *daprGrpcAPI) PublishEvent(ctx context.Context, request *runtime.Publish
 }
 
 func (d *daprGrpcAPI) GetSecret(ctx context.Context, request *runtime.GetSecretRequest) (*runtime.GetSecretResponse, error) {
-	panic("implement me")
+	if d.secretStores == nil || len(d.secretStores) == 0 {
+		err := status.Error(codes.FailedPrecondition, messages.ErrSecretStoreNotConfigured)
+		return &runtime.GetSecretResponse{}, err
+	}
+
+	secretStoreName := request.StoreName
+
+	if d.secretStores[secretStoreName] == nil {
+		err := status.Errorf(codes.InvalidArgument, messages.ErrSecretStoreNotFound, secretStoreName)
+		return &runtime.GetSecretResponse{}, err
+	}
+
+	if !d.isSecretAllowed(request.StoreName, request.Key) {
+		err := status.Errorf(codes.PermissionDenied, messages.ErrPermissionDenied, request.Key, request.StoreName)
+		return &runtime.GetSecretResponse{}, err
+	}
+
+	req := secretstores.GetSecretRequest{
+		Name:     request.Key,
+		Metadata: request.Metadata,
+	}
+
+	getResponse, err := d.secretStores[secretStoreName].GetSecret(req)
+	if err != nil {
+		err = status.Errorf(codes.Internal, messages.ErrSecretGet, req.Name, secretStoreName, err.Error())
+		return &runtime.GetSecretResponse{}, err
+	}
+
+	response := &runtime.GetSecretResponse{}
+	if getResponse.Data != nil {
+		response.Data = getResponse.Data
+	}
+	return response, nil
 }
 
 func (d *daprGrpcAPI) GetBulkSecret(ctx context.Context, request *runtime.GetBulkSecretRequest) (*runtime.GetBulkSecretResponse, error) {
```

Comment:
Besides,let me help add code for `daprGrpcAPI.GetBulkSecret`

Reply:
👌🏻

Reply:
fixed